media-libs/libvorbis/libvorbis-1.3.6.recipe

```diff
@@ -58,15 +70,29 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage libvorbis$secondaryArchSuffix \
+	"$libDir"/libvorbis.so.$libvorbisVersion \
+	"$libDir"/libvorbisenc.so.$libvorbisencVersion \
+	"$libDir"/libvorbisfile.so.$libvorbisfileVersion \
+
+PATCH()
+{
+	# use developDocDir instead of a hard-coded docdir
+	sed -i -e "s|^\(docdir = \)\$(datadir)/doc/\$(PACKAGE)-\$(VERSION)|\1\$(prefix)/$relativeDevelopDocDir|;" \
+		doc/libvorbis/Makefile.am \
+		doc/vorbisenc/Makefile.am \
+		doc/vorbisfile/Makefile.am \
+		doc/Makefile.am
+}
+
 BUILD()
 {
 	libtoolize -fci
 	mkdir -p m4
 	aclocal --install -I m4
 	autoconf
 	automake
```

Comment:
`autoreconf` could be used here to shorten the recipe.

Reply:
Well, I did try using autoreconf, but it did not work as expected, so I decided that the best thing do do was not to shorten the recipe but to limit the changes to the minimum.
BTW, I already forgot what errors I bumped into when I tried using autoreconf.